deployer.Dockerfile

```diff
@@ -4,12 +4,48 @@ FROM node:14.17.5-alpine
 
 COPY --from=solc /usr/bin/solc /usr/bin/solc
 
-WORKDIR /truffle
+WORKDIR /hardhat
 
-RUN npm i -g truffle && \
-truffle init && \
-npm i @openzeppelin/contracts@4.3.0 \
-@truffle/hdwallet-provider \
-web3@1.5.2
+RUN apk update && apk add --no-cache git
+
+#RUN npm i -g truffle && \
+#truffle init && \
+#npm i @openzeppelin/contracts@4.3.0 \
+#@truffle/hdwallet-provider \
+#web3@1.5.2
+
+RUN yarn add --dev "hardhat@^2.8.4" \
```

Comment:
@elblockdude can these dependencies be installed via the hardhat/package.json file that is already defined?

Reply:
Definitely this should be replaced with `npm i`.

Reply:
Well i'll see if that works and come back to you guys